### ISSUE TITLE
Point towards latest MBS release 

### ISSUE THREAD

**Issue by Jday7879** [ONSdigital/construction-survey-results#3]
We have set the version of MBS pipeline to install as 0.1.1.
I attempted to point this towards the latest release to avoid manually updating this file but couldn't get it to work
Another approach might be better once we have time to look into it

https://github.com/ONSdigital/construction-survey-results/blob/9fa813d1289a992ab361b8d246a9c131d7b3450f/setup.cfg#L20

**Comment by AntonZogk** [comment]
I don't think it should point to latest release at least not setup , since MBS is deployed in artifactory. But we definitely need a workaround for the workflows which they should use the last release in the github? I think there is an API to retrieve the latest release tag

**Comment by Jday7879** [comment]
Possible solutions:
Grab the latest release without hard coding. 
Workflows - if mbs present install from github instead of pypi
Add this to backlog to investigate

**Comment by AntonZogk** [comment]
Adding here because might forget, bash commands to update installation route for monthly-business-survey-results, this should be used only for the workflows and point the mbs dependency to github latest release

`latest_release=$(curl --silent -m 10 --connect-timeout 5 "https://api.github.com/repos/ONSdigital/monthly-business-survey-results/releases/latest"  | 
grep '"tag_name":' |                                          
sed -E 's/.*"([^"]+)".*/\1/') `
 
 
`perl -pi -e "s,monthly-business-survey-results,monthly-business-survey-results @ git+https://github.com/ONSdigital/monthly-business-survey-results.git@$release,g" setup.cfg`

reference : 
https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28
https://stackoverflow.com/questions/525592/find-and-replace-inside-a-text-file-from-a-bash-command
